app/models/event.rb

```diff
@@ -9,4 +9,11 @@ class Event < ApplicationRecord
   def has_member?(user)
     members.include?(user)
   end
+
+  def event_status
+    return '' unless self.started_at
+    return 'in_progress' if self.started_at == Time.now
+    return 'incomming' if self.started_at > Date.today
+    return 'finished' if self.started_at < Date.today
+  end
```

Comment:
The very important thing here is missing translations. This will be working only for English users. But because you cannot use I18n in model you need to find another way to allow change translations.

Reply:
Should I use enum here?

Reply:
Rails enum is connected to db fields. You can use symbols here, but because we need some logic here it is better to stay with normal method.

Reply:
I added one more comment about translation for `index.html`